templates/python/custom-copilot-basic/env/.env.dev.user.tpl

```diff
@@ -0,0 +1,13 @@
+# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
+
+# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
+SECRET_BOT_PASSWORD=
+{{#useOpenAI}}
+SECRET_OPENAI_API_KEY=<openai-api-key>
```

Comment:
Unlike the samples in Teams-AI repo, since we will only have OpenAI or AzureOpenAI envs in this file, how do you think about leaving the value empty? So Toolkit can throw MissingEnvironmentVariable error if user forget to fill the value. This may make the troubleshooting easier.

Reply:
In this template, I changed judgement and Toolkit will throw Runtime error.